src/pages/home/report/ReportActionsList.js

```diff
@@ -335,6 +335,12 @@ function ReportActionsList({
         (reportAction, index) => {
             let shouldDisplay = false;
             if (!currentUnreadMarker) {
+                // Check if the report type is "REPORTREVIEW" and last actor is the current user.
+                // Return shouldDisplay new marker action (terminate flow).
+                // This is to avoid displaying the new line marker when a current userrequests money.
+                if (ReportActionsUtils.isReportPreviewAction(reportAction) && reportAction.childLastActorAccountID === Report.getCurrentUserAccountID()) {
+                    return shouldDisplay;
```

Comment:
I think we should not return from here as the logic below should be followed by Money requests

Reply:
o nice, I think that works too